itest/zero_value_anchor_test.go

```diff
@@ -290,7 +290,7 @@ func testZeroValueAnchorAccumulation(t *harnessTest) {
 	require.NoError(t.t, t.tapd.stop(false))
 
 	// Enable sweeping in the config.
-	t.tapd.clientCfg.Wallet.SweepZeroValueAnchorUtxos = true
+	t.tapd.clientCfg.Wallet.SweepOrphanUtxos = true
```

Comment:
oh didn't know we had such direct access to the config

Why not do this for `testZeroValueAnchorSweep` too? This way we can get rid of `tapdOptions` on the test case itself

Reply:
Unless mistaken, the `tapd` instance automatically created when starting the test would need to be stopped, the config changed and restarted for this to work, which are operations unrelated to the test itself (not like here where we actually want to do this restart as part of the test itself).